scripts/src/admin/services.ts

```diff
@@ -698,13 +698,23 @@ type UpdateAppConfigRequest = Request & {
 };
 
 export const updateAppConfig = async function(req: UpdateAppConfigRequest, res: Response) {
+	const config = req.body;
+	if (
```

Comment:
i'm not a fan of this formatting at all, why not:
```
const predicate = (limitName: string) => typeof config.limits[limitName] === "number";
if (!config.limits || !Object.keys(config.limits).every(predicate)) {
    ...
```

also, maybe check that the `limitName` values match those we have, and that all of them have been passed?

Reply:
Done
"maybe check that the `limitName` values match those we have" - what does it mean?